modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/PortalImplTest.java

```diff
@@ -62,6 +68,31 @@ public class PortalImplTest {
 	public static final LiferayIntegrationTestRule liferayIntegrationTestRule =
 		new LiferayIntegrationTestRule();
 
+	@Test
+	public void testGetCanonicalURLWithURLSeparatorInFriendlyURL()
```

Comment:
the test is failiing on CI ? can you review it?

Reply:
Yes, I'm on it. I'll close this PR and send it later with fixes.